provider/googleProvider.js

```diff
@@ -64,7 +64,11 @@ class GoogleProvider {
           } else {
             // support for API calls with arbitraty deepness
             filArgs.reduce((p, c) => p[c], this.sdk)(requestParams, (error, response) => {
-              if (error) reject(new Error(error));
+              if (error && error.errors && error.errors[0].message && error.errors[0].message.includes('project 1043443644444')) {
```

Comment:
Is `1043443644444 ` a Google error code?

Reply:
`1043443644444` is the Id number of `my-project`.

From what I understand, a project name is unique across all GCP, and each project had an Id number. We put `my-project` in the `serverless.yml` for the GCF template. If you don't change that, it will spit an error about project 1043443644444 not being configured correctly.